ote/src/cljs/ote/app/controller/transport_service.cljs

```diff
@@ -11,6 +11,16 @@
             [ote.localization :refer [tr tr-key]]
             [ote.app.controller.place-search :as place-search]))
 
+(defn clean-app-state [app]
+      (let [transport-service (get app :transport-service)
```

Comment:
Why is this function needed? It seems convoluted, can't we just reset service to `nil` or empty map `{}`.

Reply:
Maybe the function name is confusing. This cleans up transport-service object for the form. But it doesn't remove it completely. 
I will rename it.